jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -187,9 +193,17 @@ public void initColumns(final ListTable<ProcessInstanceSummary> extendedPagedTab
         extendedPagedTable.getColumnSortList().push(startColumn);
     }
 
-    public void addDomainSpecifColumns(ExtendedPagedTable<ProcessInstanceSummary> extendedPagedTable,
-                                       Set<String> columns) {
+    protected boolean isSameColumn(GridColumnPreference gridColumnPreference, List<ColumnMeta<ProcessInstanceSummary>> columns) {
```

Comment:
Method could be moved to _AbstractMultiGridView_ for reuse.
Also, would recommend renaming to something like _existsColumnWithSameName_ to make it more clear.

Reply:
ok